### PR TITLE
Reserving a large block of strings for statistics

### PR DESCRIPTION
**Pull request by thyeggman**
Just so I don't have to fiddle with them later.  See #4306

### CODE REVIEW

**Comment by IntelOrca** [comment]
Could we trim down the number of statistics we record here, it just seems a bit extreme to record things like the number of banked turns built or the number times you click the rotate button. I can't see why those figures would interest anyone.

File: data/language/en-GB.txt [hunk 1]
```diff
@@ -4228,6 +4228,164 @@ STR_5916    :{COMMA16} player
...
+STR_5953    :Guests Followed
+STR_5954    :Highest Guest Number
+STR_5955    :Guest Deaths
+STR_5956    :Guests Combusted
+STR_5957    :Accidental Deaths
+STR_5958    :Guests Drowned
+STR_5959    :Trash Thrown Away
+STR_5960    :Squares Painted
+STR_5961    :Hill Tool Uses
+STR_5962    :Volume of Land Moved (M^3)
```
Comment (ThePsionic) [inline data/language/en-GB.txt]: small m

**Comment by Goddesen** [comment]
@IntelOrca I didn't look at the entire list, but stuff like "Times rotate button clicked" is the kind of statistic I find neat. Not for any real purpose other than "damn these devs keep track of everything!", but I like it 👍

**Comment by IntelOrca** [comment]
There will come a point though where we just have statistic increment lines filling up the codebase.

**Comment by ThePsionic** [comment]
@IntelOrca a statistic for "times a statistic increased"? ;P

**Comment by thyeggman** [comment]
If you decide that not all the statistics are useful, I can leave some out.  Some of these I expect to be harder to implement.  In any case, I just wanted to add strings for all the ones that I thought of.

**Comment by IntelOrca** [comment]
We will wait for some evidence of implementation of statistics before adding so many strings to en-gb.

**Pull request closed without merging**